File: .ai/install/project-templates/store-pilot/backend/api/create_store.py
from backend.application.create_store import create_store


def handle_create_store(payload: dict, store_id: str) -> dict:
    store = create_store(store_id, str(payload.get("name") or ""))
    return {"id": store.id, "name": store.name, "status": store.status}

File: .ai/install/project-templates/store-pilot/frontend/src/create-store.ts
import type { CreateStoreRequest, Store } from "../../contracts/generated/sdk/contracts";

export async function createStore(request: CreateStoreRequest): Promise<Store> {
  const response = await fetch("/stores", {
    method: "POST",
    headers: { "content-type": "application/json" },
    body: JSON.stringify(request),
  });
  if (!response.ok) throw new Error(`create store failed: ${response.status}`);
  return response.json() as Promise<Store>;
}
